internal/mount/mount_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package mount
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseBlkidCommand(t *testing.T) {
+	blk := &Blk{
+		DevName:   "/dev/nvme1n1p1",
+		UUID:      "f3e74cbb-4b6c-442b-8cd1-d12e3fc7adcd",
+		BlockSize: 1,
+		Type:      "ntfs",
+	}
+
+	blocks := []struct {
+		outFormatString string
+		Error           bool
+	}{
+		{
+			outFormatString: "DEVNAME=%s\nUUID=%s\nBLOCK_SIZE=%d\nTYPE=%s\n",
+			Error:           false,
+		},
+		{
+			outFormatString: "No-Equal-Sign\n",
+			Error:           true,
+		},
+		{
+			outFormatString: "BLOCK_SIZE=isString\n",
+			Error:           true,
+		},
+	}
+
+	for _, block := range blocks {
```

Comment:
You can improve the test by adding names to the test cases and executing them as subtests, see https://github.com/edgelesssys/contrast/blob/78f04de12d7bf89f03fdb5548ae2ebfb2cac7873/internal/logger/handler_test.go#L54 or other unit tests in the repo.